FIO/local-volumes/dbench-job-generator-local-volume-replica.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+set -euo pipefail
+
+#
+# The following script provisions a local volume with one replica and runs fio tests
+# to measure StorageOS performance. The FIO tests that are run can be found
+# here: https://github.com/storageos/dbench/blob/master/docker-entrypoint.sh
+#
+# In order to successfully execute the tests you will need to have:
+#  - Kubernetes cluster with a minium of 3 nodes and 30 Gib space
+#  - kubectl in the PATH - kubectl access to this cluster with
+#    cluster-admin privileges - export KUBECONFIG as appropriate
+#  - StorageOS CLI running as a pod in the cluster
+#  - jq in the PATH
+#
+# Deploy the StorageOS CLI as a container:
```

Comment:
We ask the user to read this comment and then deploy the cli in all of these scripts. Can we come up with an idempotent incantation that checks if the cli is running and deploys it if not? Could defer to a separate script at the top of the repo if needed.